netcore/CoreFX.issues.rsp

```diff
@@ -836,6 +845,9 @@
 ##  System.Reflection.Emit.Lightweight.Tests
 ####################################################################
 
+-nomethod System.Reflection.Emit.Tests.DynamicMethodctor1.String_Type_TypeArray_Type
+-nomethod System.Reflection.Emit.Tests.DynamicMethodctor1.String_Type_TypeArray_Module
+
 -nomethod System.Reflection.Emit.Tests.DynamicMethodGetILGenerator1.GetILGenerator_Int_Module_CoreclrIgnoresSkipVisibility
 -nomethod System.Reflection.Emit.Tests.DynamicILInfoTests.Test_TwoDimTest
 -nomethod System.Reflection.Emit.Tests.DynamicMethodToString.ToStringTest
```

Comment:
Why the fixed tests are not removed from the exclusion list?

Reply:
Those were unrelated failing SRE tests, I fixed them by backporting this CoreCLR PR https://github.com/dotnet/coreclr/pull/24937